netmiko/plugin.spec.yaml

```
@@ -81,39 +81,45 @@ connection:
     - ubiquiti_edgeswitch
     - vyatta_vyos
     - vyos
+    example: linux
   credentials:
     title: Credentials
     type: credential_username_password
-    description: User to run commands as
+    description: Username and password to run commands as
     required: true
+    example: '{"username": "user@example.com", "password": "mypassword"}'
   secret:
     title: Secret
     type: credential_secret_key
-    description: Secret password
+    description: API seret key
```

@r7-kszczepanskagorna  Typo
```suggestion
    description: API secret key
```

and need to regenerate